### PR TITLE
Feature/233 Add Group E transaction type page

### PR DESCRIPTION
**Pull request by dheitzer**

### CODE REVIEW

File: schema/JF_TRAN.json [hunk 1]
```diff
@@ -295,19 +296,18 @@
...
-      "type": "integer",
-      "minimum": 0,
-      "maximum": 99999999,
-      "examples": [
-        20120615
-      ],
+      "type": "string",
+      "minLength": 8,
+      "maxLength": 8,
+      "pattern": "^\\d{8}$",
```
Comment (toddlees) [inline schema/JF_TRAN.json:302]: Can we make this match #87 so we don't accidentally overwrite the changes from that branch.
https://github.com/fecgov/fecfile-validate/blob/c9566940cd902cd2258b643cba0fabb4de3dc722/schema/JF_TRAN.json#L262

File: schema/JF_TRAN.json [hunk 2]
```diff
@@ -295,19 +296,18 @@
...
+      "examples": [20120615],
       "fec_spec": {
         "COL_SEQ": 21,
         "FIELD_DESCRIPTION": "CONTRIBUTION DATE",
         "TYPE": "NUM-8",
         "REQUIRED": "X (error)",
         "SAMPLE_DATA": 20120615,
-        "VALUE_REFERENCE": "YYYYMMDD",
+        "VALUE_REFERENCE": null,
```
Comment (toddlees) [inline schema/JF_TRAN.json:310]: @mjtravers is it our intention to have the fec_spec property match the original fec spreadsheet or our update one?

File: schema/JF_TRAN.json (hunk 1 above)
Comment (mjtravers) [reply schema/JF_TRAN.json]: Looks like the contribution_date was updated to the YYYY-MM-DD format so resolving this comment

File: schema/JF_TRAN.json (hunk 2 above)
Comment (toddlees) [reply schema/JF_TRAN.json:276]: Awesome! I'll keep that in mind next time I edit a schema

**Pull request merged**